app/[locale]/blog/[slug]/page.tsx

```
@@ -0,0 +1,130 @@
+              style={{ 
+                backgroundImage: `url(${post.coverImage})`,
+                backgroundSize: 'cover',
+                backgroundPosition: 'center'
+              }}
+            />
+          </div>
+          
+          {/* Article Content */}
+          <div 
+            className="prose dark:prose-invert lg:prose-lg max-w-none mb-16"
+            dangerouslySetInnerHTML={{ __html: post.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)